lldb/source/Plugins/Trace/intel-pt/CommandObjectTraceStartIntelPT.cpp

```diff
@@ -78,7 +78,7 @@ bool CommandObjectThreadTraceStartIntelPT::DoExecuteOnThreads(
     llvm::ArrayRef<lldb::tid_t> tids) {
   if (Error err = m_trace.Start(tids, m_options.m_ipt_trace_size,
                                 m_options.m_enable_tsc, m_options.m_psb_period))
-    result.SetError(Status(std::move(err)));
+    result.SetError(Status::FromError(std::move(err)));
```

Comment:
just make sure that you really need to invoke std::move. It's possible that the compiler can do that on its own and that you don't need the explicit std::move